nextflow.config

```diff
@@ -15,14 +15,23 @@ profiles {
             includeConfig "configs/slurm.config"
             process.executor = "slurm"
         }
+    
+    test_local { 
+            includeConfig 'configs/test_local.config'
+        }
 }
 
+includeConfig 'configs/modules.config'
+
 params {
 
     outdir = "/hps/nobackup/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_v6"
     dummy_fasta = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/dummy.fasta"
     empty_file = "/hps/software/users/rdf/metagenomics/service-team/users/chrisata/taxdb_generation_nf/assets/EMPTY.txt"
```

Comment:
Those 2 files and `outdir` should be moved to and output dir should be moved somewhere from Chris's folder. We should assign a general location for that (and I know those are not your fixes, just noticed because nobody reviewed tax_db generation)
@chrisAta  makes sense to review it I think

Reply:
ah yeah this was written before I started using nf-core guidelines... I will fix all of this after this PR gets merged